includes/admin/upgrades/upgrade-functions.php

```diff
@@ -1135,6 +1153,69 @@ function give_v1813_update_donor_user_roles_callback() {
 	}
 }
 
+/**
+ * Correct currency code for "Iranian Currency" for all of the payments.
+ *
+ * @since 1.8.14
+ */
+function give_v1814_update_donation_iranian_currency_code() {
+
+	/* @var Give_Updates $give_updates */
+	$give_updates = Give_Updates::get_instance();
+
+	// Payment query.
+	$payment = new WP_Query( array(
+		'paged'          => $give_updates->step,
+		'status'         => 'any',
+		'post_type'      => array( 'give_payment' ),
+		'posts_per_page' => 20,
+	) );
+
+	if ( $payment->have_posts() ) {
+		$give_updates->set_percentage( $payment->found_posts, ( $give_updates->step * 20 ) );
+
+		while ( $payment->have_posts() ) {
+			$payment->the_post();
+			global $post;
+
+			$payment_meta = give_get_meta( $post->ID, '_give_payment_meta', true );
+
+			if ( ! empty( $payment_meta ) ) {
+
+				if ( 'RIAL' === $payment_meta['currency'] ) {
+					$payment_meta['currency'] = 'IRR';
+					update_post_meta( $post->ID, '_give_payment_meta', $payment_meta );
+				}
+			}
+		}
+
+		/* Restore original Post Data */
+		wp_reset_postdata();
+	} else {
+		// The Update Ran.
+		give_set_upgrade_complete( 'v1814_update_donation_iranian_currency_code' );
+	}
+}
+
+/**
+ * Correct currency code for "Iranian Currency" in Give setting.
+ *
+ * @since 1.8.14
+ */
+function give_v1814_update_core_iranian_currency_code() {
```

Comment:
@emgk This should be in automatic updates. Can you update this?
https://github.com/WordImpress/Give/blob/release/1.8.14/includes/admin/upgrades/upgrade-functions.php#L26

Reply:
@ravinderk I have updated the code, could you please check?